cove/fixtures/tenders_releases_2_releases_codelists.json

```diff
@@ -0,0 +1,115 @@
+                        "url": "https://buyandsell.gc.ca/cds/public/2014/04/11/f3f3993b4f6f260e057d0cb95f43741d/14-2015_itt_amd2.pdf", 
+                        "id": "https://buyandsell.gc.ca/cds/public/2014/04/11/f3f3993b4f6f260e057d0cb95f43741d/14-2015_itt_amd2.pdf"
+                    }, 
+                    {
+                        "url": "https://buyandsell.gc.ca/cds/public/2014/04/11/69359ba194d3f01f4f13cb9673b85071/14-2015_itt_amd2-f.pdf", 
+                        "id": "https://buyandsell.gc.ca/cds/public/2014/04/11/69359ba194d3f01f4f13cb9673b85071/14-2015_itt_amd2-f.pdf"
+                    }
+                ], 
+                "tenderPeriod": {
+                    "endDate": "2014-04-15T14:00:00.00Z"
+                }, 
+                "items": [
+                    {
+                        "id": "1", 
+                        "classification": {
+                            "scheme": "GSIN", 
+                            "description": "F029: Other Animal Care and Control Services"
+                        }, 
+                        "description": "Canine Control of the Canada Geese (14-2015)"
+                    }
+                ], 
+                "awardCriteriaDetails": null, 
+                "procuringEntity": {
+                    "name": "Agriculture & Agrifood Canada",
+                    "name_fr": "Agriculture & Agrifood Canada"
+                }, 
+                "id": "PW-14-00627094", 
+                "methodRationale": "Open"
+            }, 
+            "tag": ["tender", "oh no"], 
+            "buyer": {
+                "name": "Agriculture & Agrifood Canada"
+            }
+        }, 
+        {
+            "date": "2014-04-04T00:00:00.00Z", 
+            "language": "English", 
+            "id": "Buyandsell.gc.ca-2014-11-07-a76b0dd2-3dc8-4c52-bc36-cd0e42530602", 
+            "ocid": "PW-14-00629344", 
+            "initiationType": "tender", 
+            "tender": {
+                "documents": [
+                    {
+                        "url": "https://buyandsell.gc.ca/cds/public/2014/04/04/cb21d6397174fbba48c92edb694de76f/tender_document_5p404-13181.pdf", 
+                        "id": "https://buyandsell.gc.ca/cds/public/2014/04/04/cb21d6397174fbba48c92edb694de76f/tender_document_5p404-13181.pdf"
+                    }, 
+                    {
+                        "url": "https://buyandsell.gc.ca/cds/public/2014/04/04/370b7a9ab2bdd35086e6caed9b5226f9/fr_tender_document_5p404-13181.pdf", 
+                        "id": "https://buyandsell.gc.ca/cds/public/2014/04/04/370b7a9ab2bdd35086e6caed9b5226f9/fr_tender_document_5p404-13181.pdf"
+                    }
+                ], 
+                "tenderPeriod": {
+                    "endDate": "2014-04-22T00:00:00Z"
+                }, 
+                "items": [
+                    {
+                        "id": "1", 
+                        "classification": {
+                            "scheme": "GSINS", 
```

Comment:
That was added in order to make a codelist error as GSINS in not in the codelist file (but GSIN is), so is there on purpose.